tabular/src/autogluon/tabular/trainer/model_presets/presets.py

```diff
@@ -27,6 +28,7 @@
     GBM=70,
     CAT=60,
     NN=50,
+    TRANSF=50,
```

Comment:
TRANSF should probably be lowest priority model for now until it's been extensively validated

Reply:
Changed to TRANSF=0.